include/seqan3/alignment/pairwise/execution/execution_handler_parallel.hpp

```diff
@@ -99,15 +99,14 @@ class execution_handler_parallel
 
     /*!\brief Takes underlying range of sequence pairs and invokes an alignment on each instance.
      * \tparam algorithm_t              The type of the alignment algorithm.
```

Comment:
Comment like this will be changed later?

Reply:
yes this will be changed. The plan would be to move it to core as well, since it is used by other algorithms. When moving I will also update the documentation. This is just a separate commit for the first part that generalises the two way executor.